modules/API/auth_consumer_API/docs/index.rst

```diff
@@ -8,4 +8,4 @@ auth_consumer_API
 
 :ref:`Link <everest_modules_auth_consumer_API>` to the module's reference.
 
-See ``doc/everest_api_specs/auth_consumer_API/asyncapi.yaml`` for a full AsycAPI specification.
\ No newline at end of file
+See ``doc/everest_api_specs/auth_consumer_API.yaml`` for a full AsycAPI specification.
```

Comment:
This path (and others) should now be ``docs/source/reference/EVerest_API/auth_consumer_API.yaml``. It would be nice if we could directly to the generated html here